### ISSUE TITLE
Unable to get sample working when updating to Angular 9(.1.9)

### ISSUE THREAD

**Issue by jafin** [davidwalschots/angular-reactive-validation#8]
Good afternoon,

First off apologies if I use incorrect terminology, I am green in the area of Angular.

I've attempted first in my project, and now with this project's sample to update the app to Angular 9.  I get receiving an error from the sample app that 

The app kept errors when calling getFormControlFromContainer()  due to controlContainer.control being null and throwing an exception.  No validation works at all.

```javascript
core.js:1 ERROR TypeError: Cannot read property 'get' of null
    at FormGroupDirective.getFormGroup (forms.js:7151)
    at FormGroupName.get control [as control] (forms.js:6203)
    at getFormControlFromContainer (angular-reactive-validation.js:7)
    at ValidationMessageComponent.set for [as for] (angular-reactive-validation.js:7)
    at setInputsFromAttrs (core.js:1)
    at instantiateAllDirectives (core.js:1)
    at createDirectivesInstances (core.js:1)
    at Module.ɵɵelementStart (core.js:17040)
    at AppComponent_Template (app.component.html:14)
    at executeTemplate (core.js:1)
```

So I moved the logic that was in

```typescript
set for(control: FormControl | string | undefined) {
    this._for = typeof control === 'string' ? getFormControlFromContainer(control, this.controlContainer) : control;
    this._forControl = control;
  }
```
to onInit, where controlContainer.control does not appear to be null.

```typescript
set for(control: FormControl | string | undefined) {
    this._forControl = control;
}

  ngOnInit(){
    this._for = typeof this._forControl === 'string' ? getFormControlFromContainer(this._forControl, this.controlContainer) : this._forControl;
  }

```

I then attempt the demo project again and receive the error:
```javascript
errors.ts:30 ERROR Error: Cannot find control with name: 'age'
    at _throwError (shared.ts:141)
    at setUpControl (shared.ts:36)
    at FormGroupDirective.addControl (form_group_directive.ts:138)
    at FormControlName._setUpControl (form_control_name.ts:227)
    at FormControlName.ngOnChanges (form_control_name.ts:150)
    at FormControlName.wrapOnChangesHook_inPreviousChangesStorage (ng_onchanges_feature.ts:75)
    at callHook (hooks.ts:248)
    at callHooks (hooks.ts:214)
    at executeInitAndCheckHooks (hooks.ts:159)
    at selectIndexInternal (advance.ts:70)
```

If I remove the name validators and try again it succeeded.
Or if I remove the age validators the name validators succeed.  But not both at once.
So I don't think the change I did was correct.

Does anyone have any suggestions on getting the sample app working in Angular 9?

**Comment by davidwalschots** [comment]
@jafin I'm assuming that you were trying to work on Angular 9 compatibility with a commit before my [latest changes](https://github.com/davidwalschots/angular-reactive-validation/commit/0d9dcaf0b0ffc03c1b1c4bd2f78fb2e2f34b251f).

I've just published these changes as version 4.0.1 of the package. It should now be fully compatible!

Let me know if you run into any problems.

**Comment by jafin** [comment]
Yes apologies I didn't even notice the Angular 9 branch! I'll run with the 4.0.1 package.  Thanks!
